crossing-training.lic

```diff
@@ -671,9 +671,12 @@ class CrossingTraining
       check_osrel unless @researching
       check_research
       retreat
-      loop do
+      attempts_to_hide = 5
+      while attempts_to_hide > 0 do
         break if hide?
+        attempts_to_hide - 1
```

Comment:
This doesn't assign back to `attempts_to_hide`, so I think this will still be an infinite loop... can you double check?